numpy_financial/_financial.py

```diff
@@ -571,12 +572,19 @@ def pv(rate, nper, pmt, fv=0, when='end'):
 #  p*((r + 1)^n - 1)*(r*w + 1)/r^2 + n*p*(r + 1)^(n - 1)*(r*w + 1)/r +
 #  p*((r + 1)^n - 1)*w/r)
 
+
 def _g_div_gp(r, n, p, x, y, w):
+    # Evaluate g(r_n)/g'(r_n), where g =
+    # fv + pv*(1+rate)**nper + pmt*(1+rate*when)/rate * ((1+rate)**nper - 1)
     t1 = (r+1)**n
     t2 = (r+1)**(n-1)
-    return ((y + t1*x + p*(t1 - 1)*(r*w + 1)/r) /
-                (n*t2*x - p*(t1 - 1)*(r*w + 1)/(r**2) + n*p*t2*(r*w + 1)/r +
-                 p*(t1 - 1)*w/r))
+    g = y + t1*x + p*(t1 - 1) * (r*w + 1) / r
+    gp = (
```

Comment:
Minor style quibble: Do you have a reason for the line break here?  The four terms are short enough that I think the style shown at https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator would look better.

Reply:
I thought it looked nice to have the beginning of each expression aligned (this is also done in the example), while keeping the indentation a multiple of 4. The alternatives would either mis-align the top expression

```python
    gp = (n*t2*x
        - p*(t1 - 1) * (r*w + 1) / (r**2)
        + n*p*t2 * (r*w + 1) / r
        + p*(t1 - 1) * w/r)
```
or, if I add spaces to align that expression, contain 10 spaces, which then looks awkward with every other indentation being 4/8/12 spaces.

I'm not too concerned with the either style change, let me know if you want me to fix it.

Reply:
10 spaces is fine.  That sort of alignment is pretty typical.

Reply:
Fixed. Is this fine to merge otherwise?